lib/index.js

```diff
@@ -224,6 +228,32 @@ async function analyzeProjectDependencies(options, pkgData, pkgFile) {
   return output
 }
 
+/** Get peer dependencies from installed packages */
+function getPeerDependencies(current, options) {
+  const basePath = options.cwd || './'
+  return Object.keys(current).map(pkgName => {
+    const path = basePath + 'node_modules/' + pkgName + '/package.json'
+    try {
+      const pkgData = fs.readFileSync(path)
+      const pkg = jph.parse(pkgData)
+      return vm.getCurrentDependencies(pkg, { dep: 'peer', filter: options.filter, reject: options.reject })
+    }
+    catch (e) {
+      print(options, 'Could not read peer dependencies for package ' + pkgName + '. Is this package installed?', 'warn')
+      return {}
+    }
+  }).reduce((acc, peers) => {
+    Object.entries(peers).forEach(([pkgName, version]) => {
+      if (acc[pkgName] === undefined) {
+        acc[pkgName] = []
+      }
+      // eslint-disable-next-line fp/no-mutating-methods
```

Comment:
Note: PR's that disable the linter will not be accepted in order to maintain the project's code style and quality requirements. You'll have to find a way to do this in a more functional style with no mutations.

Reply:
To be honest, I just copied that from your own code: https://github.com/raineorshine/npm-check-updates/search?q=no-mutating-methods

But not problem, I've fixed that now :-)

Reply:
Fair enough! There is some legacy code from an older version that had different linter settings.